person.rb

```diff
@@ -1,9 +1,10 @@
+require_relative 'nameable'
 # Person class
 class Person
```

Comment:
- [ ] Great job 👍 . Kindly make sure your `Person` class inherits from `Nameable` class as stated in the project requirements 🙏